runtime/devicescript/devs_value.h

```diff
@@ -6,8 +6,8 @@ typedef union {
     double _f;
     uint64_t u64;
     struct {
-        uint32_t mantisa32;
-        uint32_t mantisa20 : 20;
+        uint32_t mantissa32;
+        uint32_t mantissa20 : 20;
```

Comment:
this is a fairly common misspelling -- so common that Google doesn't suggest fixing it.